packages/fxa-shared/oauth/constants.js

```diff
@@ -5,6 +5,7 @@
 module.exports = {
   OAUTH_SCOPE_OLD_SYNC: 'https://identity.mozilla.com/apps/oldsync',
   OAUTH_SCOPE_SESSION_TOKEN: 'https://identity.mozilla.com/tokens/session',
+  OAUTH_SCOPE_NEWSLETTERS: 'https://identity.mozilla.com/account/newsletters',
```

Comment:
@dannycoates How about it?  Include this in the scopes of the token returned after a passwordless user sign up?

Reply:
sure

Reply:
hmm I will change that 'account' to 'accounts' to be consistent with the other scopes.